gpt_pp/ai/__init__.py

```diff
@@ -89,15 +89,15 @@ def get_chat(self, files: str) -> ConversationChain:
             memory=memory,
         )
 
-    @Halo(text="Generating review comments", spinner="dots")
+    @Halo(text="Reviewing pull request", spinner="dots")
```

Comment:
The updated spinner text in line 97 reflects the action being performed. Well done!